tests/tests/helpers.rs

```diff
@@ -249,3 +254,112 @@ pub async fn pipe_from_stream_and_drop<T: Serialize>(
 		}
 	}
 }
+
+#[allow(dead_code)]
```

Comment:
I'd have to compile this to check but I wonder why all the `#[allow(dead_code)]`s are needed!

Reply:
I don't remember but probably that is just used for tests are not exported outside the crate.

Maybe pub(crate) would do it...

Reply:
If they are only used for tests then having the helpers be behind `#[cfg(test)]` would also remove such a warning :)

Reply:
we don't have a lib.rs in the crate it's like https://github.com/paritytech/jsonrpsee/tree/master/tests/tests but every file should #![cfg(test)] so I don't know ^^

Reply:
Sorry about that, totally missed it